timed/employment/models.py

```diff
@@ -39,6 +42,31 @@ class Employment(models.Model):
     start_date       = models.DateField()
     end_date         = models.DateField(blank=True, null=True)
 
+    def clean(self):
+        """Validate the employment as a whole.
```

Comment:
As http://www.django-rest-framework.org/topics/3.0-announcement/#differences-between-modelserializer-validation-and-modelform describes rest framework doesn't call clean and expects validation to happen on the Serializer. Only Forms call clean.

As this code base exposes a rest api but on the other hand also does server side rendering (django admin) it can get fairly confusing to do the validation on the model level but the api actually doesn't use this validation.

As a cleaner approach I therefore recommend to move the django admin validation to the Form.clean method of the django admin which needs it.
See here how to overwrite form: https://docs.djangoproject.com/en/dev/ref/contrib/admin/#django.contrib.admin.ModelAdmin.form